### PR TITLE
Update of dependencies.

### PR DESCRIPTION
**Pull request by 766974616c79**

### CODE REVIEW

File: bruteforce/src/lib.rs [hunk 1]
```diff
@@ -2,19 +2,11 @@
...
-    generator_trait,
-    const_if_match,
-    const_panic,
     proc_macro_hygiene
 )]
 #![cfg_attr(not(feature = "std"), no_std)]
-
-extern crate test;
-
-#[cfg(not(feature = "std"))]
```
Comment (deeprobin) [inline bruteforce/src/lib.rs:17]: Why do you want to remove this line?
We only want to replace `std` with `no_std_compat` if `no_std`.

**Review by deeprobin — COMMENTED** [review]
Thank you for your pull request. I have one more comment but otherwise this looks very good.

Thanks for your efforts.

File: bruteforce/src/lib.rs (hunk 1 above)
Comment (766974616c79) [reply bruteforce/src/lib.rs:17]: For me, it doesn't make much sense

**Comment by deeprobin** [comment]
@MoskalykA CI / Lint is failing. Please fix your code style.

**Comment by 766974616c79** [comment]
It is good :)

**Review by deeprobin — APPROVED** [review]
LGTM

**Comment by deeprobin** [comment]
Thank you for the contribution 🎉.

**Pull request merged**